### PR TITLE
enh(acl) add poller access acl config

### PR DESCRIPTION
**Pull request by a-launois**
## Description

Adapt ACL granularity to allow non-admin users to delete, export config, create/edit pollers.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by jeremyjaouen — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 1]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
...
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons “Ajouter”, Ajouter (avancé)” et “Dupliquer”)"
+
+msgid "Allows user to delete pollers."
+msgstr "Permet aux utilisateurs de supprimer des collecteurs"
+
+msgid "Allows user to generate and export configuration, and restart poller."
+msgstr "Permet aux utilisateurs de déployer la configuration de la supervision sur les collecteurs."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7242]: inconsistent translation

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 2]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
...
+msgstr "Créer et modifier des collecteurs"
+
+msgid "Delete pollers"
+msgstr "Supprimer des collecteurs"
+
+msgid "Deploy configuration"
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons “Ajouter”, Ajouter (avancé)” et “Dupliquer”)"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7236]: use infinitive

Comment (kduret) [reply lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7236]: not needed for consistency with other help tooltip

Comment (a-launois) [reply lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7236]: Infinitive is not use on this page

**Review by jeremyjaouen — DISMISSED** [review]

**Review by adr-mo — DISMISSED** [review]

**Review by callapa — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 3]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
 "The optimized mode will only be available and functional when using the BBDO 3.0.0 protocol for the collect engine."
 msgstr "Mode utilisé pour récupérer les données affichées dans la page Resource Status."
 "Le mode optimisé pourra être utilisé et fonctionnel que si le moteur de supervision utilise le protocole BBDO 3.0.0"
+
+msgid "Poller configuration actions / Poller management"
+msgstr "Gestion des serveurs de collecte/collecteurs"
```
Comment (smau-centreon) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7224]:
```suggestion
msgstr "Actions de configuration du collecteur / Gestion du collecteur"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: www/include/options/accessLists/actionsACL/formActionsAccess.php [hunk 4]
```diff
@@ -189,6 +191,7 @@
 $form->addElement('header', 'host_actions', _("Hosts Actions Access"));
 $form->addElement('header', 'global_actions', _("Global Monitoring Engine Actions (External Process Commands)"));
 $form->addElement('header', 'global_access', _("Global Functionalities Access"));
+$form->addElement('header', 'poller_cfg_access', _("Poller configuration actions / Poller management"));
```
Comment (a-launois) [inline www/include/options/accessLists/actionsACL/formActionsAccess.php:194]:
```suggestion
$form->addElement('header', 'poller_cfg_access', _("Poller Configuration Actions / Poller Management"));
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 5]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
...
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons Ajouter, Ajouter (avancé) et Dupliquer)."
+
+msgid "Allows user to delete pollers."
+msgstr "Permet aux utilisateurs de supprimer des collecteurs."
+
+msgid "Allows user to deploy configuration."
+msgstr "Permet aux utilisateurs de déployer la configuration de la supervision sur les collecteurs."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7242]:
```suggestion
msgstr "Permet aux utilisateurs de déployer la configuration."
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 6]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
...
+msgstr "Supprimer des collecteurs"
+
+msgid "Deploy configuration"
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons Ajouter, Ajouter (avancé) et Dupliquer)."
+
+msgid "Allows user to delete pollers."
+msgstr "Permet aux utilisateurs de supprimer des collecteurs."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7239]:
```suggestion
msgstr "Permet à l'utilisateur de supprimer des collecteurs."
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 7]
```diff
@@ -7219,3 +7219,24 @@ msgid "Mode used to retrieve information displayed in Resource Status page."
...
+msgstr "Créer et modifier des collecteurs"
+
+msgid "Delete pollers"
+msgstr "Supprimer des collecteurs"
+
+msgid "Deploy configuration"
+msgstr "Déployer la configuration"
+
+msgid "Allows user to perform “Add”, “Add (advanced)” and “Duplicate” actions on pollers."
+msgstr "Permet aux utilisateurs de créer des collecteurs (boutons Ajouter, Ajouter (avancé) et Dupliquer)."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:7236]:
```suggestion
msgstr "Permet à l'utilisateur de créer des collecteurs (boutons Ajouter, Ajouter (avancé) et Dupliquer)."
```

**Review by smau-centreon — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**